x-pack/plugins/data_enhanced/server/search/es_search_strategy.ts

```diff
@@ -57,7 +57,13 @@ export const enhancedEsSearchStrategyProvider = (
       return toAsyncKibanaSearchResponse(body);
     };
 
-    return pollSearch(search, options).pipe(
+    const cancel = async () => {
```

Comment:
Nit: We're already defining a `cancel` function below, and it would be nice to combine them since they behave the same (except the other also takes care of logging).

Reply:
(The same with the `eql_search_strategy`.)